airmozilla/starred/tests/test_views.py

```diff
@@ -227,3 +227,53 @@ def test_anonymous_starred_events(self):
         ok_(event1.title in response.content)
         ok_(event2.title in response.content)
         ok_(event3.title not in response.content)
+
+    def test_event_pagination(self):
+        url = self.home_url
+        user = self._login(username='lisa')
+
+        events = []
+
+        for i in range(25):
+            event = self.create_event("Event Title %d" % (i + 1))
+            StarredEvent.objects.create(user=user, event=event)
+            events.append(event.id)
+
+        response = self.client.get(url)
+        eq_(response.status_code, 200)
+
+        for id in range(len(events)):
```

Comment:
The `for thing in range(len(sequence)):` is known as an anti-pattern in Python. 
Almost always you should just iterate on the object itself. E.g. `for thing in sequence:`

However, if you want to the index numbers when iterating over a list you use `enumerate()`. E.g.

```python
list1 = ['a','b','c']
list2 = ['d', 'e', 'f']
for i, item in list1:
    print (item, list2[i])
```